packages/ui/src/components/atoms/Checkbox/stories/Checkbox.stories.mdx

```diff
@@ -0,0 +1,273 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs'
+
+import { Checkbox } from '@faststore/components'
+
+import {
+  TokenTable,
+  TokenRow,
+  TokenDivider,
+} from 'src/../.storybook/components'
+
+<Meta
+  title="Atoms/Checkbox"
+  component={Checkbox}
+  argTypes={{
+    disabled: {
+      type: {
+        name: 'boolean',
+      },
+      description: '`boolean`',
+      defaultValue: false,
+    },
+    partial: {
+      type: {
+        name: 'boolean',
+      },
+      defaultValue: false,
+    },
+  }}
+/>
+
+export const Template = ({ id, ...args }) => (
+  <div style={{ display: 'flex' }}>
+    <Checkbox id={id} {...args} onChange={() => {}} />
+    <label htmlFor={id} style={{ paddingLeft: '1rem' }}>
+      label
+    </label>
+  </div>
+)
+
+<header>
+
+# Checkbox
+
+An input Checkbox that is commonly used on Filter component.
+
+</header>
+
+## Overview
+
+The `Checkbox` component uses [FastStore UI Checkbox](https://www.faststore.dev/reference/ui/atoms/Checkbox) as base.
```

Comment:
For now, I'm removing this part 🙈. Or we can just replace for `Faststore Componentes` (and remove the link, since we don't have it)

Reply:
Nice. Agree.